lib/cadet/application.ex

```diff
@@ -1,4 +1,7 @@
 defmodule Cadet.Application do
+  @moduledoc """
```

Comment:
This module exist by Phoenix convention, it is expected for developers to understand what it contains, so `@moduledoc false` will do